### DISCUSSION TITLE
GitHub Advanced Security Certification Exam Prep 🧠

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#137197] [Enterprise]
_This discussion and its comment have been edited and modified from its original format to enable evergreen learning_

Want to get GitHub Advanced Security certified? The Community has created some supplementary prep so you can pass the [GitHub Advanced Security exam](https://learn.github.com/credentials) with ease. 

This will be broken up into four sections with curated resources, additional exam questions from our Hubber experts, and. Q&A. 

### Why should I get certified? 
Getting GitHub certified is a resounding endorsement that validates your skills, credibility, trust, and knowledge of the technologies and developer tools that are used by more than 100 million developers worldwide. 

### What is the GitHub Advanced Security certification? 
Highlight your code security knowledge with the GitHub Advanced Security certification. Validate your expertise in vulnerability identification, workflow security, and robust security implementation—elevating software integrity standards.

### How do I complete the prep course? 
At your own pace, we recommend doing it over a four week basis, doing one part each week. But we've created four discussions breaking down the topics covered in the GHAS certification exam. 

### What’s the breakdown? 
**Part One**

- [Getting Started](https://github.com/orgs/community/discussions/137620)
    - Intro to GitHub Advanced Security 
    - Configure Dependabot security updates on your GitHub repo
       
**Part Two**
 
- [Scanning ](https://github.com/orgs/community/discussions/138220)
   - Configure and use secret scanning in your GitHub repository
   - Configure code scanning on GitHub
   
**Part Three**
 
- [CodeQL](https://github.com/orgs/community/discussions/137620)
   - Identify security vulnerabilities in your codebase by using CodeQL
    - Code scanning with GitHub CodeQL
    
**Part Four**

   * [Managing Your GitHub Securely ](https://github.com/orgs/community/discussions/139526)
      * GitHub administration for GitHub Advanced Security
      * Manage sensitive data and security policies within GitHub

Then, once you complete the course and the exam, we’ll be asking for your feedback so we can continue to make the certification process a great one! We’re looking forward to getting you exam ready. 

### Looking for more Certifications prep in the Community? 
- [Copilot Certification Exam Prep Course](https://github.com/orgs/community/discussions/144443): The GitHub Copilot certification exam evaluates your skill in using the AI-driven code completion tool in various programming languages, certifying your capability to optimize software development workflows efficiently.

We're hard at work creating more materials to help you ace your exam. We'll be sharing these as they become available! 

For more information on our certifications program, please visit the [Certifications site](https://learn.github.com/credentials).

**Comment by queenofcorgis** [comment]
Part One's Getting Started Discussion section with prep materials and sample questions is now live - [visit the Discussion](https://github.com/orgs/community/discussions/137620) to start prepping for the GitHub Advanced Security Certification Exam!

**Comment by queenofcorgis** [comment]
We've got our foundations to GHAS now we're ready to learn all about scanning - join the [Part Two Discussion](https://github.com/orgs/community/discussions/138220) to continue studying for the exam.

**Comment by queenofcorgis** [comment]
[Part Three's Discussion](https://github.com/orgs/community/discussions/138831) is live and ready for you to continue studying 📖 . Week Three will be focusing on CodeQL!

**Comment by queenofcorgis** [comment]
Time for our final installment in this prep course. To finish studying and get fully prepped join us in [this Discussion](https://github.com/orgs/community/discussions/139526) for Part Four.

**Comment by queenofcorgis** [comment]
Looking to study along with us for another certification, [join the discussion](https://github.com/orgs/community/discussions/174475) for Week One of the GitHub Administration Certification prep course!

**Comment by admin** [comment]
love it
